bin/clone

```diff
@@ -38,29 +36,36 @@ def main():
     parser.add_argument(
         "--css-validate", help="set whether css validation is required", type=str_to_bool, default=None
     )
+    parser.add_argument("--path", help="path to save the page to be cloned", required=False, default="/opt/")
```

Comment:
If you keep /opt/ as default, you should use `sudo`. And I prefer to keep it as default. So Virtual env is optional, but not forced